### PR TITLE
Add initial setter support

### PR DESCRIPTION
**Pull request by dhylands**
r? @julienw 

I was able to use the svc.py script (from fxbox/tools/scripts) to do things like:

``` bash
2126 >./svc.py --service=Open
Adapter: OpenZwave Adapter ID: OpenZWave-db421a3b
  setters:
    OpenZWave-db421a3b-0100000002588000 (Locked)
    OpenZWave-db421a3b-0100000005494000 (Switch)
  getters:
    OpenZWave-db421a3b-0100000002588000 (Locked)
    OpenZWave-db421a3b-0100000005494000 (Switch)
2127 >./svc.py --get OpenZWave-db421a3b-0100000005494000
OpenZWave-db421a3b-0100000005494000 = 'On'
2128 >./svc.py --set OpenZWave-db421a3b-0100000005494000=Off
2129 >./svc.py --get OpenZWave-db421a3b-0100000005494000
OpenZWave-db421a3b-0100000005494000 = 'Off'
```

### CODE REVIEW

**Comment by dhylands** [comment]
This needs https://github.com/fxbox/taxonomy/pull/85 to land before this PR can land.

**Comment by dhylands** [comment]
I renamed LockedUnlocked to be DoorLocked as per review on the taxonomy side of things.

**Comment by dhylands** [comment]
The DoorLocked change to taxonomy landed.

File: src/lib.rs [hunk 1]
```diff
@@ -291,7 +291,7 @@ impl OpenzwaveAdapter {
                     ZWaveNotification::ValueAdded(value)           => {
                         if value.get_genre() != ValueGenre::ValueGenre_User { continue }
 
-                        let value_id = format!("OpenZWave/{} ({})", value.get_id(), value.get_label());
+                        let value_id = format!("OpenZWave-{:08x}-{:016x} ({})", value.get_home_id(), value.get_id(), value.get_label());
```
Comment (julienw) [inline src/lib.rs]: ah right; I remembered we had everything in ValueID but I forgot it was not all in the id. Thanks !

**Comment by julienw** [comment]
landed in https://github.com/fxbox/openzwave-adapter/pull/18

**Pull request closed without merging**